src/views/todo/subcomponents/NewPageComponent.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <div class="d-flex justify-content-center w-100">
+    <div class="d-flex flex-column w-100">
+      <div>
+        <p class="title fs-1" aria-label="page-title">New Page Title</p>
```

Comment:
@joven1export nitpick `class` or any styling attributes should be moved to the last part of the element. we do this practice because we want to be able to see the behavior and identifiers of the element, before it's styling. especially since the class attribute can take up a lot of space

for `aria-label`, this is usually used for interactive elements. I don't think this will be accessed by any screen reader technology since `p` tags don't work with it 😄 paragraph elements naturally contains the text that the screen readers will read. `aria-label` is to help screen readers understand interactive elements or icons what the use of it is for, especially if it's not written in the element itself (e.g. a button with simply a `filter` icon but no text). You can read up more about it here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label